api/manager-api.service

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do we still need this file?

Reply:
Hii, nope it's not required anymore. I forgot to clean it up.